### DISCUSSION TITLE
Is it possible to filter on untagged and unread?

### DISCUSSION THREAD

**Discussion by brhahlen** [wallabag/wallabag#7697] [Q&A]
It seems that I can only filter on _either_ untagged or unread, but not both at the same time, apparently.

Am I missing something?

**Comment by HolgerAusB** [comment]
Not with the UI. If you are in `/untagged/list` and set a filter to 'unread' you will be moved to `/all/list`
But you can change the URL manually:
`https://example.com/untagged/list?entry_filter[isUnread]=1`

**Reply by HolgerAusB** [reply]
but that is also nearly useless. as soon as I [delete an entry|read an entry and return|multi-tag some entries] from that filtered view, the filter vanishes and I am again in unfiltered `/untagged/list`
